admin/server-timing.php

```diff
@@ -168,3 +179,77 @@ class="large-text code"
 	</p>
 	<?php
 }
+
+/**
+ * Renders a checkbox for enabling output buffering for Server-Timing.
+ *
+ * @since n.e.x.t
+ */
+function perflab_render_server_timing_page_output_buffer_checkbox() {
+	$slug           = 'output_buffering';
+	$field_id       = "server_timing_{$slug}";
+	$field_name     = PERFLAB_SERVER_TIMING_SETTING . '[' . $slug . ']';
+	$description_id = "{$field_id}_description";
+	$has_filter     = has_filter( 'perflab_server_timing_use_output_buffer' );
+	$is_enabled     = perflab_server_timing_use_output_buffer();
+
+	?>
+	<style>
+	/* The following styles are copied from core where they normally include `.form-table td` in the selectors. */
+	fieldset.output-buffering {
+		margin-bottom: 9px;
+		padding: 15px 10px 15px 0;
+		line-height: 1.3;
+		vertical-align: middle;
+		font-size: 14px;
+	}
+	fieldset.output-buffering label,
+	fieldset.output-buffering p {
+		line-height: 1.4;
+	}
+	fieldset.output-buffering label {
+		margin: 0.35em 0 0.5em !important;
+		display: inline-block;
+	}
+	fieldset.output-buffering p {
+		font-size: 14px;
+	}
+	</style>
+	<fieldset class="output-buffering">
+		<legend class="screen-reader-text">
+			<?php esc_html_e( 'Output Buffering', 'performance-lab' ); ?>
+		</legend>
+		<input
+			type="checkbox"
+			id="<?php echo esc_attr( $field_id ); ?>"
+			name="<?php echo esc_attr( $field_name ); ?>"
+			aria-describedby="<?php echo esc_attr( $description_id ); ?>"
+			<?php disabled( $has_filter ); ?>
+			<?php checked( $is_enabled ); ?>
+		>
+		<label for="<?php echo esc_attr( $field_id ); ?>">
+			<?php esc_html_e( 'Enable output buffering of template rendering.', 'performance-lab' ); ?>
+		</label>
+		<p id="<?php echo esc_attr( $description_id ); ?>" class="description">
+			<?php if ( $has_filter ) : ?>
+				<?php if ( $is_enabled ) : ?>
+					<?php
+					echo wp_kses(
+						__( 'Output buffering has been forcibly enabled via the <code>perflab_server_timing_use_output_buffer</code> filter.', 'performance-lab' ),
+						array( 'code' => array() )
+					);
+					?>
+				<?php else : ?>
+					<?php
+					echo wp_kses(
+						__( 'Output buffering has been forcibly disabled via the <code>perflab_server_timing_use_output_buffer</code> filter.', 'performance-lab' ),
+						array( 'code' => array() )
+					);
+					?>
```

Comment:
Related to my other comment, _if_ we don't put the other text below into an `else`, we should add a `<br>` here to avoid overly long lines.